Index: base/parse2/NumberNames.hs
===================================================================
RCS file: /cvs/programatica/tools/base/parse2/NumberNames.hs,v
retrieving revision 1.5
diff -u -r1.5 NumberNames.hs
--- base/parse2/NumberNames.hs	8 Dec 2001 01:18:40 -0000	1.5
+++ base/parse2/NumberNames.hs	2 Jun 2005 16:47:07 -0000
@@ -7,5 +7,5 @@
 --
 numberNames m = mapNames conv m
   where
-    conv (SN i p) = PN i (S p)
+    conv (SN i p) = PN i (Sn (show i) p)
 --}
