--- register.sh	2005-06-14 11:38:22.000000000 +0200
+++ register-file.sh	2005-06-14 11:41:30.000000000 +0200
@@ -24,8 +24,8 @@
                  HTMLMonad HTMLMonadAttrs HTMLMonad98 HTMLMonadBase HTMLPrelude
                  HTMLPrelude98 HTMLTypedBase HTMLTypedBase98 HTMLTemplates
 hidden-modules:
-import-dirs: /usr/local/lib/WASH-2.3.1
-library-dirs: /usr/local/lib/WASH-2.3.1
+import-dirs: '$HOME'/wash-pkg/lib/WASH-2.3.1
+library-dirs: '$HOME'/wash-pkg/lib/WASH-2.3.1
 hs-libraries: HSWASH-2.3.1
 extra-libraries:
 include-dirs:
@@ -37,4 +37,4 @@
 framework-dirs:
 frameworks:
 haddock-interfaces:
-haddock-html:' | /home/linux-bkb/bin/ghc-pkg --auto-ghci-libs update
+haddock-html:' | /home/linux-bkb/bin/ghc-pkg --auto-ghci-libs update "$@"
